apps/native-component-list/package.json

```
@@ -115,7 +115,7 @@
     "expo-sensors": "~12.0.0",
     "expo-sharing": "~11.1.0",
     "expo-sms": "~11.1.0",
-    "expo-speech": "~11.0.0",
+    "expo-speech": "~11.1.0",
```

Please do not bump versions manually, we have a separate package publishing process and this is not a scope of this PR
```suggestion
    "expo-speech": "~11.0.0",
```

---

ok my bad. the builds were failing without this so I thought I had to do it. I'll revert.